FunctionWithArguments

```diff
@@ -0,0 +1,13 @@
+/*
+
+1. Create a function called functionWithArgs that accepts two arguments and   outputs their sum to the dev console.
+2. Call the function with two numbers as arguments.
```

Comment:
file names dont have an extension why?

Reply:
I created JS files in VSCode, where it recognized JS files, I thought it assigned the extension. I will make sure not to repeat this.